projects/lib/src/lib/services/luigi-nodes/local-configuration.service.ts

```diff
@@ -44,26 +49,52 @@ export class LocalConfigurationServiceImpl
       const configurations = await this.getLocalConfigurations(
         localDevelopmentSettings
       );
-      const luigiNodes =
-        await this.luigiConfigService.getLuigiNodesFromConfigurations(
+      const result =
+        (await this.luigiConfigService.getLuigiNodesFromConfigurations(
           configurations
-        );
+        )) || {};
+
+      if (result.errors) {
+        this.alertErrors(result.errors);
+      }
 
-      (luigiNodes || []).forEach((node) => {
+      (result.nodes || []).forEach((node) => {
         node.context = {
           ...node.context,
           serviceProviderConfig: localDevelopmentSettings.serviceProviderConfig,
         };
       });
 
-      this.cachedLocalNodes = luigiNodes;
-      return luigiNodes;
+      return (this.cachedLocalNodes = result.nodes || []);
     } catch (e) {
       console.warn(`Failed to retrieve local luigi config.`, e);
       return [];
     }
   }
 
+  private alertErrors(errors: ValidationResult[]) {
+    const message = errors
+      .map((e) => {
+        return `For configuration with url:
+            ${e.url} <br/><br/>
```

Comment:
based on the interface url can be undefined, if so the we should not show undefined but rather avoid showing the url and only display the errors
```suggestion
            ${e.url} <br/><br/>
```

Reply:
the url required, and is always there https://github.com/openmfp/portal-server-lib/blob/fbb41b1c32bac1c22d280c6a9714d25544b93607/src/config/luigi/luigi-data/content-configuration-validator.service.ts#L57